java-checks/src/test/files/checks/StaticImportCountCheck/MixedStandardAndStaticImports.java

```diff
@@ -0,0 +1,9 @@
+import java.util.Date;
+import java.util.Collections;
+import java.util.List;
+
+import static java.lang.Math.*;
+import static java.util.Collections.*;
+import static com.myco.corporate.Constants.*;
+import static com.myco.division.Constants.*;
+import static com.myco.department.Constants.*;  // Noncompliant
```

Comment:
If I want to report whole line, should I use [[sc=1]] ? It works, but I didn't find any other test which uses it this way